.github/labeler.yml

```diff
@@ -0,0 +1,48 @@
+batch:
+  - '(batch)'
+bug:
+  - '(bug)'
+config.writer:
+  - '(config.writer)'
+documentation:
+  - '(documentation)'
+duplicate:
+  - '(duplicate)'
+enhancement:
+  - '(enhancemet)'
+gempyor:
+  - '(gempyor)'
+good first issue:
+  - '(good first issue)'
+help wanted:
+  - '(help wanted)'
+high priority:
+  - '(high priority)'
+inference:
+  - '(inference)'
+invalid:
+  - '(invalid)'
+low priority:
+  - '(low priority)'
+medium priority:
+  - '(medium priority)'
+meta/workflow:
+  - '(meta/workflow)'
```

Comment:
```yml
  - '(meta|workflow)'
```

Reply:
This is a particularly good catch on your part lol. Such a thorough PR reviewer! Really appreciate it.

Reply:
Mostly a function of having written many bad regexes in the past 😅. Np!